examples/tina-self-hosted-demo/content/pages/home.md

```diff
@@ -30,11 +30,107 @@ blocks:
   - tagline: This is a new tagline!
     headline: This is a test
     text: >
-      This project is set up to show you the basics of working with Tina. You're
-      looking at the landing page, which pulls content from
+      # dsad
```

Comment:
**question**: Was `dsad` meant to be committed?

Reply:
True - we should keep the content not completely random :D